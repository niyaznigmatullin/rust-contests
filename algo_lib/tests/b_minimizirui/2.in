7
1 7 2 4 8 4 100
7
1 3 6 3
2 2 7
1 2 3 5
2 1 7
1 1 7 3
2 1 4
2 2 7
