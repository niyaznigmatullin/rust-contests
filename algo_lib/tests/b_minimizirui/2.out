118
117
9
17
